tests/src/psa_exercise_key.c

```diff
@@ -707,20 +688,28 @@ int mbedtls_test_psa_exported_key_sanity_check(
     else
 #endif /* MBEDTLS_ECP_C */
 
-    if( PSA_KEY_TYPE_IS_PUBLIC_KEY( type ) )
     {
-        char message[47];
-        mbedtls_snprintf( message, sizeof( message ),
-                          "No sanity check for public key type=0x%08lx",
-                          (unsigned long) type );
-        mbedtls_test_fail( message, __LINE__, __FILE__ );
-        return( 0 );
+        TEST_ASSERT( ! "Sanity check not implemented for this key type" );
     }
-    else
 
+#if defined(MBEDTLS_DES_C)
```

Comment:
It is reachable. DES is considered an unstructured type. (In retrospect this may or may not have been the right thing: it helps about as often as it hurts.)